packages/core/src/Abstracts/TxBuilderAbstract.class.ts

```diff
@@ -0,0 +1,15 @@
+import { AssetAmount, IAssetAmountMetadata } from "@sundaeswap/asset";
+import { EDatumType, IComposedTx, IPoolData } from "src/@types";
```

Comment:
Make sure you update all "src/" paths to be relative and module extensions: i.e. `"../src/@types/index.js`. Not doing so will cause issues with compilers downstream.

Reply:
Fixed and found to the setting in vscode to avoid this :)